src/tvheadend/entity/Tag.cpp

```diff
@@ -90,7 +90,7 @@ bool Tag::ContainsChannelType(bool bRadio) const
   {
     if ((cit = channels.find(*it)) != channels.end())
     {
-      if (bRadio == cit->second.IsRadio())
+      if (cit->second.GetType() == (bRadio ? CHANNEL_TYPE_RADIO : CHANNEL_TYPE_TV))
```

Comment:
If bRadio is false, the function returns true for tv channels only..
But changing the function to what you suggest makes sense.